dg/urls.py

```diff
@@ -69,19 +69,16 @@
 from django.contrib.auth.models import User
 from rest_framework import routers, serializers, viewsets
 from people.models import Person
+from geographies.models import Country,Village
+import people.urls 
+
+from people.views import VillagesViewSet, FarmerViewSet, FarmerDetailView, BlockViewSet
+
 # Serializers define the API representation.
 class UserSerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
Add class docstring

Reply:
Code is outdated but same applied to other classes in the project